07-http_gordle/final/internal/domain/errors.go

```diff
@@ -0,0 +1,14 @@
+package domain
```

Comment:
Maybe `session` would make more sense as the package name, it's basically describing a single user game over the span of time.